apps/services/routers.py

```diff
@@ -289,6 +289,35 @@ async def read_user_document(
     )
 
 
+@services_router.get("/{university_id}/download-user-document/{user_document_id}",
+                     name="download_user_document",
+                     response_class=FileResponse,
+                     summary="Download user document",
+                     responses={200: {
+                         "description": "Successful download user document response",
+                         "content": {"text/html": {"example": "file"}}
+                     }},
+                     tags=["Student dashboard"])
+async def download_user_document(
```

Comment:
Hi @victor-nuzhniy 
I can't open the downloaded file if I run the application inside the docker container. Can you please fix it?
Please see examples in ticket comments

Reply:
Fixed it